core/src/integrationTest/kotlin/ch/ergon/dope/helper/BaseIntegrationTest.kt

```diff
@@ -0,0 +1,71 @@
+package ch.ergon.dope.helper
+
+import ch.ergon.dope.DopeQuery
+import ch.ergon.dope.helper.DBProvider.cluster
+import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.resolvable.fromable.UnaliasedBucket
+import ch.ergon.dope.validtype.ArrayType
+import ch.ergon.dope.validtype.BooleanType
+import ch.ergon.dope.validtype.NumberType
+import ch.ergon.dope.validtype.StringType
+import com.couchbase.client.kotlin.query.QueryParameters
+import com.couchbase.client.kotlin.query.QueryScanConsistency
+import com.couchbase.client.kotlin.query.execute
+import kotlinx.coroutines.runBlocking
+import java.time.Instant.now
+import kotlin.time.Duration.Companion.seconds
+
+abstract class BaseIntegrationTest {
+    private val maxTimeout = 15.seconds
```

Comment:
extract to a constant and give it a more precise name. Like _CLUSTER_TIMEOUT_ or something

Reply:
done